oss/build_whl.sh

```diff
@@ -91,11 +91,12 @@ function main() {
   $PYTHON_BIN -c 'import array_record'
   $PYTHON_BIN -c 'from array_record.python import array_record_data_source'
   # TF is not available on Python 3.13 and above.
-  if [ "$(uname)" != "Darwin" ] && (( "${PYTHON_MINOR_VERSION}" < 13 )); then
+  # if [ "$(uname)" != "Darwin" ] && (( "${PYTHON_MINOR_VERSION}" < 13 )); then
+  if (( "${PYTHON_MINOR_VERSION}" < 13 )); then
```

Comment:
Once https://github.com/google/grain/pull/954 is merged and Grain gets a pre-release we can actually have this change.